py_v_sdk/contract/atomic_swap_ctrt.py

```diff
@@ -162,3 +184,230 @@ async def get_tok_bal(self, addr: str) -> int:
             int: The balance of the token.
         """
         return await self._query_db_key(self.DBKey.for_token_balance(addr))
+
```

Comment:
For each state var & state map, a method or property like `get_tok_bal` should be added so that users can easily query the db.